cockatrice/src/interface/widgets/tabs/tab_deck_editor.cpp

```diff
@@ -164,16 +182,18 @@ void TabDeckEditor::showPrintingSelector()
     printingSelectorDockWidget->setVisible(true);
 }
 
+/**
+ * @brief Loads deck editor layout from settings or resets to default.
+ */
 void TabDeckEditor::loadLayout()
 {
     LayoutsSettings &layouts = SettingsCache::instance().layouts();
-
     setCentralWidget(databaseDisplayDockWidget);
 
     auto &layoutState = layouts.getDeckEditorLayoutState();
-    if (layoutState.isNull()) {
```

Comment:
what's the reason for removing these braces? I don't think removing them increases readability here